perllib/FixMyStreet/Roles/Cobrand/KingstonSutton.pm

```diff
@@ -119,17 +119,20 @@ sub waste_munge_report_form_fields {
 =head2 waste_report_form_first_next
 
 After picking a service, we jump straight to the about you page unless it's
-bulky, where we ask for more information.
+bulky or small items, where we ask for more information.
 
 =cut
 
 sub waste_report_form_first_next {
     my $self = shift;
     my $cfg = $self->feature('echo');
-    my $bulky_service_id = $cfg->{bulky_service_id};
+    my $bulky_or_small_service_id
+        = $cfg->{bulky_service_id} || $cfg->{small_items_service_id};
```

Comment:
Sutton already has bulky, so this will always be bulky - so I think this won't do what you're after for a small items collection, because bulky won't be ticked? The reason the test works if because it's not setting bulky_service_id.